tests/unit/lib/Gocdb_Services/APIAuthenticationServiceTest.php

```diff
@@ -172,29 +172,31 @@ public function testGetAPIAuthentication()
         $type = 'X.509';
       // Start with no APIAuthentication entities to be found
         $this->assertNull(
-            $authEntServ->getAPIAuthentication($ident, $type),
+            $authEntServ->getAPIAuthentication($ident),
             "Non-null value returned when searching for APIAuthentication entity " .
-            "for id:{$ident} with type:{$type} when expected none."
+            "for id:{$ident} when expected none."
         );
 
         $authEnt = $siteService->addAPIAuthEntity(
             $site,
             $user,
-            array('IDENTIFIER' =>  $ident,
-                              'TYPE' => $type,
-            'ALLOW_WRITE' => false)
+            array(
+                'IDENTIFIER' =>  $ident,
+                'TYPE' => $type,
```

Comment:
Where will `$type` come from?

Reply:
Line 172, it's not been removed when adding an API Identifier so that the approporiate validation can be run against it.